doc/working_groups_raw_data.yaml

```diff
@@ -17,6 +17,8 @@ workgroups:
       - 'Peter Norton'
       - 'Vernon Cole'
       - 'Prashanth Goud B'
+      - 'Tyler Johnson'
+      - 'Michael Verhulstm'
```

Comment:
This should just be `'Michael Verhulst'`, with no final `m`. Same change below in the README.

I work with him :)

Reply:
Thanks :) Changed it now